index.html

```diff
@@ -0,0 +1,117 @@
+          </p>
+          <p>
+            Lorem ipsum dolor sit amet, consectetur adipiscing elit. Vivamus quis tempus massa. Aenean erat nisl, gravida vel vestibulum cursus, interdum sit amet lectus. Sed sit amet quam nibh. Suspendisse quis tincidunt nulla. In hac habitasse platea dictumst. Ut sit amet pretium nisl. Vivamus vel mi sem. Aenean sit amet consectetur sem. Suspendisse pretium, purus et gravida consequat, nunc ligula ultricies diam, at aliquet velit libero a dui.
+          </p>
+          <ul>
+            <li></li>
+          </ul>
+        </article>
+      </section>
+    </section>
+    <section class="main-section" id="theme">
+      <header>theme</header>
+      <section>
+        <article>
+          <p>
+            <code>
+              
+            </code>
+          </p>
+          <p>
+            Lorem ipsum dolor sit amet, consectetur adipiscing elit. Vivamus quis tempus massa. Aenean erat nisl, gravida vel vestibulum cursus, interdum sit amet lectus. Sed sit amet quam nibh. Suspendisse quis tincidunt nulla. In hac habitasse platea dictumst. Ut sit amet pretium nisl. Vivamus vel mi sem. Aenean sit amet consectetur sem. Suspendisse pretium, purus et gravida consequat, nunc ligula ultricies diam, at aliquet velit libero a dui.
+          </p>
+          <ul>
+            <li></li>
+          </ul>
+        </article>
+      </section>
+    </section>
+    <section class="main-section" id="Rating">
+      <header>Rating</header>
+      <section>
+        <article>
+          <p>
+            <code>
+              
+            </code>
+          </p>
+          <p>
+            Lorem ipsum dolor sit amet, consectetur adipiscing elit. Vivamus quis tempus massa. Aenean erat nisl, gravida vel vestibulum cursus, interdum sit amet lectus. Sed sit amet quam nibh. Suspendisse quis tincidunt nulla. In hac habitasse platea dictumst. Ut sit amet pretium nisl. Vivamus vel mi sem. Aenean sit amet consectetur sem. Suspendisse pretium, purus et gravida consequat, nunc ligula ultricies diam, at aliquet velit libero a dui.
+          </p>
+          <ul>
+            <li></li>
+          </ul>
+        </article>
+      </section>
+    </section>
+    <section class="main-section" id="Reference">
+      <header>Reference</header>
+      <section>
+        <article>
+          <p>
+            <code>
+              
+            </code>
+          </p>
+          <p>
+            Lorem ipsum dolor sit amet, consectetur adipiscing elit. Vivamus quis tempus massa. Aenean erat nisl, gravida vel vestibulum cursus, interdum sit amet lectus. Sed sit amet quam nibh. Suspendisse quis tincidunt nulla. In hac habitasse platea dictumst. Ut sit amet pretium nisl. Vivamus vel mi sem. Aenean sit amet consectetur sem. Suspendisse pretium, purus et gravida consequat, nunc ligula ultricies diam, at aliquet velit libero a dui.
+          </p>
+          <ul>
+            <li></li>
+          </ul>
+        </article>
+      </section>
+    </section>
```

Comment:
Kindly I recommend you remove blank lines, spaces, or indentations from your work.

Reply:
- [ ] Why do you want the student to remove blank lines, please alwaysexplain why a change is necessary. This helps the student learn from each code review and grow their skills too.